packages/connection-form/src/hooks/use-connection-color.ts

```diff
@@ -83,15 +126,38 @@ export function legacyColorsToColorCode(
 export function useConnectionColor(): {
   connectionColorCodes: () => ColorCode[];
   connectionColorToHex: (colorCode: string | undefined) => string | undefined;
+  connectionColorToHexActive: (
+    colorCode: string | undefined
+  ) => string | undefined;
   connectionColorToName: (colorCode: string | undefined) => string | undefined;
 } {
   const newColorCodeToHex = useCallback(
     (colorCode: string | undefined): string | undefined => {
+      const isDarkMode = useDarkMode();
```

Comment:
We probably should move this hook outside of this function and have it just once for both this useCallback and the one below it.